src/client/components/LastMessageList/LastMessagesList.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable camelcase */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { object } from '@storybook/addon-knobs';
```

Comment:
I dont think you should add the story book knobs in the normal component. Pretty sure that is only for the storybook component. Or am i wrong?

Reply:
I remove that but it was just in lastMessagesList